.github/workflows/vite_hardhat.yaml

```diff
@@ -29,5 +29,8 @@ jobs:
       - name: Install dependencies
         run: bun install
 
+      - name: Compile
+        run: bunx hardhat compile
+
```

Comment:
adding this step is pointless. Compilation is done ad-hoc before tests. Same for the other workflow file

Reply:
Had another look and this is true when running `hardhat test`. But when using `bun test` (visibly faster), it has no hooks